src/util/evalUnits.ts

```diff
@@ -14,8 +14,10 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 const si = [
-  { value: -1e24, symbol: 'y' },
-  { value: -1e21, symbol: 'z' },
+  // Deactivate these huge nums for now
+  // > than 1e21 is too big for JS to handle
+  // { value: -1e24, symbol: 'y' },
```

Comment:
Can we use big num instead?

Reply:
Alter all parts of code and now we use BN